README.md

```diff
@@ -21,108 +21,103 @@ Add this to your `bb.edn` or `deps.edn`:
 Sample babashka usage:
 ```console
 $ bb -Sdeps '{:deps {io.github.lispyclouds/bblgum {:git/sha "7ebae0e2231899fe2a6ad44bc9ef5fca64099fcd"}}}' \
-     -e "(require '[bblgum.core :as b]) (b/gum {:cmd :input :opts {:placeholder \"User name:\"}})"
+     -e "(require '[bblgum.core :as b]) (b/gum :input :placeholder \"User name:\")"
 ```
 
 ## Interaction
 
 This follows the same [section](https://github.com/charmbracelet/gum#interaction) on the gum repo and all params should work verbatim.
 Run `gum <cmd> --help` to discover all the params and args.
 
-This lib only has _one_ public fn: `bblgum.core/gum`. This is possibly the tiniest clojure lib!
```

Comment:
bit sad about this going away, but will make my peace with it 🙏🏽

Reply:
wait